src/tls_wrap.cc

```diff
@@ -663,6 +663,7 @@ void TLSWrap::OnReadSelf(ssize_t nread,
   Local<Object> buf_obj;
   if (buf != nullptr)
     buf_obj = Buffer::New(wrap->env(), buf->base, buf->len).ToLocalChecked();
+  fprintf(stderr, "%d\n", nread);
```

Comment:
what's going on here? accidental debug or something else, same with the `return 0` below.